python/test/unit/language/test_core_amd.py

```diff
@@ -2917,6 +2917,71 @@ def test_convert2d(dtype, shape, src_layout, dst_layout, device='cuda'):
     assert torch.equal(z, x)
 
 
+layouts = [
+    BlockedLayout([2, 2], [16, 4], [2, 2], [1, 0], [1, 1], [1, 1], [0, 1]),
+    BlockedLayout([2, 2], [16, 4], [2, 2], [0, 1], [1, 1], [1, 1], [0, 1]),
+    BlockedLayout([1, 8], [16, 4], [4, 1], [1, 0], [1, 1], [1, 1], [0, 1]),
+    BlockedLayout([1, 8], [16, 4], [4, 1], [0, 1], [1, 1], [1, 1], [0, 1]),
+]
+
+@pytest.mark.parametrize("M, N, M_tile_size, N_tile_size, M_tile_offset, N_tile_offset", [[256, 256, 128, 64, 64, 128], [128, 128, 128, 32, 0, 0], [128, 128, 128, 32, 0, 64]])
```

Comment:
I tried to add 
`[256, 128, 256, 32, 0, 0]`
And fixed all {M}x{M} issues in the IR, but there are mismatches for this config. Can you take a look?